src/DataViews/SamplingReportDataView.cpp

```diff
@@ -450,4 +455,62 @@ SampledFunction& SamplingReportDataView::GetSampledFunction(unsigned int row) {
   return functions_[indices_[row]];
 }
 
+ErrorMessageOr<void> SamplingReportDataView::WriteStackEventsToCsv(const std::string& file_path) {
```

Comment:
What about a test for this function?

Reply:
As promised, tests will be added in the next PR.